test/e2e/hca-helpers.js

```diff
@@ -58,8 +58,8 @@ function completeMilitaryService(client, data) {
   client
     .selectDropdown('root_lastServiceBranch', data.lastServiceBranch)
     .fillDate('root_lastEntryDate', data.lastEntryDate)
-    .selectDropdown('root_dischargeType', data.dischargeType)
-    .fillDate('root_lastDischargeDate', data.lastDischargeDate);
+    .fillDate('root_lastDischargeDate', data.lastDischargeDate)
+    .selectDropdown('root_dischargeType', data.dischargeType);
```

Comment:
Swapping these around like this is probably okay, but if it was timing out before, I don't know if this will help. :man_shrugging:

Reply:
I think this is actually a bug in the code website's. If you fill out the discharge type dropdown first. Then type the discharge date in, after typing the first two digits of the year the discharge type dropdown is hidden then reappears when the final digits of the year are added while losing its set value. It seems like the show/hide shouldn't be toggled until the year is has four digits (doubt there are many vets born in 980...).

Reply:
Yeah, we could probably adjust that. You can't continue if you enter a year before 1900, so it doesn't end up causing any problems, but it's a little janky.